CondFormats/Luminosity/src/classes_def.xml

```diff
@@ -6,4 +6,5 @@
  <class name="std::vector< lumi::HLTInfo >"/>
  <class name="std::vector< lumi::TriggerInfo >"/>
  <class name="std::vector< lumi::BunchCrossingInfo >"/>
+ <class name="LumiCorrections" class_version="0"/>
```

Comment:
I don't know about this one. @samhiggie made this line.  Where did you get it from?

Reply:
Hi, I copied the instructions almost exactly from this twiki example to register an object to the DB. https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCondObjectsTutorial#The_object_class 
They have this format in the classes_def.xml... maybe it's not needed and this is an old format. I don't really know.

Reply:
indeed, I don't know either - i asked @ggovi to provide instructions...

Reply:
By the way: I also noticed it, but I did not comment because I realized that all classes in the CondFormats are have  class_version="0", see
https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_9_4_X_2017-10-15-2300&_filestring=CondFormats&_string=class_version&_casesensitive=1

Good to try to understand if it does correspond to a kind of (justified) rule.

Reply:
It looks like a lot of people were following the twiki.  :)  IMO this may be superfluous but shouldn't harm anything.  Merging this PR shouldn't be held up by understanding this issue which is really unrelated (and ignored up to now in other places).  It just looks like the twiki should be updated by someone ( @ggovi ?).

I don't mind changing this, but the approval of this PR shouldn't linger for days because of this.

Reply:
the class version is no longer needed for the persistency since ORA disappeared ( cmssw_7_2_X )

Reply:
the class version is no longer needed for the persistency since ORA disappeared ( cmssw_7_2_X )

Reply:
thanks @ggovi - maybe your documentation and existing class_versions can get cleaned up in a followup PR.